commands/backups/restore.js

```diff
@@ -0,0 +1,88 @@
+'use strict'
+
+const co = require('co')
+const cli = require('heroku-cli-util')
+
+function dropboxURL (url) {
+  // Force a dump file to download instead of rendering as HTML
+  // file by specifying the dl=1 param for Dropbox URL
+  if (url.match(/www.dropbox.com/) && !url.endsWith('dl=1')) {
```

Comment:
Should we do more principled filtering here, though? This will match things like `https://example.com?q=www.dropbox.com`. I suppose it's not likely to actually have false-positive matches in the real world, though, so maybe we can leave it to future cleanup.